documentation/suggested-project-structure.md

```diff
@@ -0,0 +1,97 @@
+### Suggested Project Structure
+
+ * `server.js` is the main entry point.
+ * Think of the `resources` folder as your `routes`.
+ * Think of the `handlers` folder as your `controllers`.
+
+```
+├── handlers
+│   ├── articleHandler.js
+│   ├── commentHandler.js
+│   ├── peopleHandler.js
+│   ├── photoHandler.js
+│   └── tagHandler.js
+├── resources
+│   ├── articles.js
+│   ├── comments.js
+│   ├── people.js
+│   ├── photos.js
+│   └── tags.js
+└── server.js          
+```
+
+#### Example server.js
+
+This is the main entry point for your API. The goal is to configure jsonapi-server and load all the resources.
+
+```javascript
+var jsonApi = require("jsonapi-server");
+var fs = require("fs");
+var path = require("path");
+
+jsonApi.setConfig({
+  // Put your config here!
+});
+
+// Load all the resources
+fs.readdirSync(path.join(__dirname, "/resources")).filter(function(filename) {
```

Comment:
I understand the reasoning behind this, but those 5 require statements would read nicer and be easier to understand. Since this is only a demo and not a production piece of code I still feel this is overkill, but I won't block this PR on it.